src/Input.js

```diff
@@ -0,0 +1,38 @@
+class Input {
+  static isValidGuess(userInput) {
+    if (userInput.length !== 3) throw new Error("3자리 숫자를 입력해주세요.");
+
+    const userInputArr = Array.from(userInput);
+    userInputArr.every((char) => this.checkNumber(Number(char)));
+    userInputArr.every((char) => this.checkZero(Number(char)));
+
+    const numbers = new Set();
+    userInputArr.forEach((number) => {
+      numbers.add(number);
+    });
+    if (numbers.size < 3) throw new Error("중복되지 않는 숫자를 입력해주세요.");
```

Comment:
```suggestion
    if (new Set(userInputArr) < 3) throw new Error("중복되지 않는 숫자를 입력해주세요.");
```
`Set`의 생성자가 `iterable`을 받으니 이렇게 줄일 수 있을 것 같습니다!

Reply:
`Set의 생성자가 iterable을 받는다.`

깔끔한 코드 👍